report_element_visibility/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Report Element Visibility
+=========================
+
+This module allows you to add 2 elements in QWEB reports:
+ - not-firt-page: Shows element in every page but first
+ - not-last-page: Shows element in every page but last
+
+Usage
+=====
+
+To use this module, you need to:
+
+Install the module, then in the QWEB ``ir.ui.views`` used by your
+report, you can add an element with css class with not-first-page or not-last-page.
+For exemple if i need to improve invoice report header with
+invoice's number in every page but first, and sale order report header 
+with order's name in every page but last, i'll add this code to external_layout_header::
+
+    <t t-if="o._table=='account_invoice'">
+        <div class="not-first-page">
```

Comment:
@pedrobaeza is_not_first_page() is a python or javascript method?
We have use this solution because with python we don't know how to get current page number, and because we think that a css class is more user's friendly.